### ISSUE TITLE
How to change username

### ISSUE THREAD

**Issue by okiirawan** [louislam/uptime-kuma#1252]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Can someone help me how to change username?

### 🐻 Uptime-Kuma Version

1.11.x

### 💻 Operating System and Arch

Ubuntu 20.04

### 🌐 Browser

Chrome

**Comment by koen20** [comment]
I don't think that's possible currently in the UI. This is a duplicate of #1243

**Comment by egooner** [comment]
If you are familiar with a bit of SQL I just had this same problem. Backed up my monitors. Shutdown Uptime Kuma and manually updated the record in the DB. update user set username = new username where id = currentuserid;
Restarted Uptime Kuma and logged in with the new username.

**Comment by g-vdn** [comment]
> 

Worked liked a charm for me, thanks

**Comment by AugusDogus** [comment]
Why was this closed as completed @louislam? #1243 is indeed a duplicate issue, but a solution was never provided there either.

**Comment by rikvdlooi** [comment]
Easy way to do this when running in docker:

Get a bash shell in the container (replace `container-name` with the name of your container):

```shell
docker exec -it container-name /bin/bash
```

Now inside the container get a command line shell for sqlite:

```shell
sqlite3 data/kuma.db
```

In the sqlite command line shell update the username of your user

```sqlite
update user set username = 'new-username' where id = 1;
```

To exit the sqlite command line shell type `.exit`. Then hit `ctrl` + `c` to exit the bash shell of the container.

I did this with a running container, although I made sure to logout first. Not sure if that is even necessary.
